### PR TITLE
Added more shields to README

### PR DESCRIPTION
**Pull request by dbrgn**
Yay for shields!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Rust Space API Implementation
 
-[![Build Status](https://travis-ci.org/coredump-ch/spaceapi-server-rs.svg?branch=master)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
+[![Travis branch](https://img.shields.io/travis/coredump-ch/spaceapi-server-rs/master.svg)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
+[![Crates.io](https://img.shields.io/crates/v/spaceapi-server.svg)](https://crates.io/crates/spaceapi-server)
+[![Crates.io](https://img.shields.io/crates/d/spaceapi-server.svg)](https://crates.io/crates/spaceapi-server)
```
Comment (rnestler) [inline README.md]: Why twice?

Comment (dbrgn) [inline README.md]: Yep :)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Rust Space API Implementation
 
-[![Build Status](https://travis-ci.org/coredump-ch/spaceapi-server-rs.svg?branch=master)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
+[![Travis branch](https://img.shields.io/travis/coredump-ch/spaceapi-server-rs/master.svg)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
```
Comment (rnestler) [inline README.md]: What's the difference between the travis links? And why rename `Build Status` into `Travis branch`

Comment (dbrgn) [inline README.md]: That's just the title text. I simply copied the template from shields.io. There's no difference besides small details in design.

**Pull request merged**